cala-ledger-core-types/src/journal.rs

```diff
@@ -0,0 +1,12 @@
+use serde::{Deserialize, Serialize};
+
+use super::primitives::*;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct JournalValues {
+    pub id: JournalId,
+    pub name: String,
+    pub external_id: Option<String>,
```

Comment:
in sqlx-ledger we also have a ```version ``` field i was unsure about what it is used for, can add that as well. 
[link](https://github.com/GaloyMoney/sqlx-ledger/blob/main/ledger/src/journal/entity.rs#L12)

Reply:
That's fine - we will introduce it via `events.len()` - no need for it as a field.